python: rpy2 - warnings.warn('pandas >= 1.0 is not supported.') - ERROR rpy2/tests/ipython/test_ggplot.py - rpy2.rinterface_lib.embedded.RRunti...

**Describe the bug**
A clear and concise description of what the bug is.

```
============================= test session starts ==============================
platform linux -- Python 3.7.9, pytest-5.4.3, py-1.9.0, pluggy-0.13.1
rootdir: /build/rpy2-3.2.6
collected 692 items / 1 error / 691 selected                                   

==================================== ERRORS ====================================
______________ ERROR collecting rpy2/tests/ipython/test_ggplot.py ______________
rpy2/tests/ipython/test_ggplot.py:3: in <module>
    import rpy2.robjects.lib.ggplot2
rpy2/robjects/lib/ggplot2.py:57: in <module>
    lazyeval = importr('lazyeval', on_conflict='warn')
rpy2/robjects/packages.py:477: in importr
    env = _get_namespace(rname)
rpy2/rinterface_lib/conversion.py:40: in _
    cdata = function(*args, **kwargs)
rpy2/rinterface.py:789: in __call__
    raise embedded.RRuntimeError(_rinterface._geterrmessage())
E   rpy2.rinterface_lib.embedded.RRuntimeError: Error in loadNamespace(name) : there is no package called ‘lazyeval’
E   Calls: <Anonymous> ... loadNamespace -> withRestarts -> withOneRestart -> doWithOneRestart
------------------------------- Captured stderr --------------------------------
R[write to console]: Error in loadNamespace(name) : there is no package called ‘lazyeval’
Calls: <Anonymous> ... loadNamespace -> withRestarts -> withOneRestart -> doWithOneRestart

=============================== warnings summary ===============================
rpy2/robjects/pandas2ri.py:14
  /build/rpy2-3.2.6/rpy2/robjects/pandas2ri.py:14: FutureWarning: pandas.core.index is deprecated and will be removed in a future version.  The public classes are available in the top-level namespace.
    from pandas.core.index import Index as PandasIndex

rpy2/robjects/pandas2ri.py:34
  /build/rpy2-3.2.6/rpy2/robjects/pandas2ri.py:34: UserWarning: pandas >= 1.0 is not supported.
    warnings.warn('pandas >= 1.0 is not supported.')

-- Docs: https://docs.pytest.org/en/latest/warnings.html
=========================== short test summary info ============================
ERROR rpy2/tests/ipython/test_ggplot.py - rpy2.rinterface_lib.embedded.RRunti...
!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!
========================= 2 warnings, 1 error in 5.99s =========================
builder for '/nix/store/sp2533fydvzb3hdz414rkv3j3wvc39ri-python3.7-rpy2-3.2.6.drv' failed with exit code 2
cannot build derivation '/nix/store/p4kjkz23jj4p70wk5cmwq1576y26xkpq-python3-3.7.9-env.drv': 1 dependencies couldn't be built
error: build of '/nix/store/p4kjkz23jj4p70wk5cmwq1576y26xkpq-python3-3.7.9-env.drv' failed

```

**To Reproduce**
Steps to reproduce the behavior:
```
with import <nixpkgs> {};
let
  envPy = python37.withPackages ( ps: with ps; [
    ipython 
    jupyterlab
    rpy2
  ]
  )
  ;
in mkShell {
  buildInputs = [
    (envPy)
    black
  ];
}
```


**Expected behavior**
A clear and concise description of what you expected to happen.

Install rpy2 with 
which R shows to path 
and import rpy2 gets imported

**Notify maintainers**
<!--
Please @ people who are in the `meta.maintainers` list of the offending package or module.
If in doubt, check `git blame` for whoever last touched something.
-->
@joelmo

**Metadata**
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```
 - system: `"x86_64-linux"`
 - host os: `Linux 5.8.13, NixOS, 20.09beta992.7badbf18c45 (Nightingale)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3.7`
 - channels(root): `"nixos-20.09beta992.7badbf18c45"`
 - channels(user): `"nixos-20.09beta992.7badbf18c45"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`

```

--- a/pkgs/development/python-modules/rpy2/default.nix
+++ b/pkgs/development/python-modules/rpy2/default.nix
@@ -25,13 +25,13 @@
 }:
 
 buildPythonPackage rec {
-    version = "3.2.6";
+    version = "3.3.6";
     pname = "rpy2";
 
     disabled = isPyPy;
     src = fetchPypi {
       inherit version pname;
-      sha256 = "1p990cqx3p2pd1rc9wn66m56wahaq8dlr88frz49vb7nv4zw4a8q";
+      sha256 = "0xvfkxvh01r5ibd5mpisp8bz385hgpn27b988y8v65z7hqr3y1nf";
     };
 
     buildInputs = [
@@ -53,6 +53,7 @@ buildPythonPackage rec {
       DBI
       dbplyr
       hexbin
+      lazyeval
       lme4
       tidyr
     ]) ++ extraRPackages ++ rWrapper.recommendedPackages;
@@ -69,14 +70,6 @@ buildPythonPackage rec {
       # R_LIBS_SITE is used by the nix r package to point to the installed R libraries.
       # This patch sets R_LIBS_SITE when rpy2 is imported.
       ./rpy2-3.x-r-libs-site.patch
-
-      # pandas 1.x compatibility, already merged upstream
-      # https://github.com/rpy2/rpy2/issues/636
-      (fetchpatch {
-        name = "pandas-1.x.patch";
-        url = "https://github.com/rpy2/rpy2/commit/fbd060e364b70012e8d26cc74df04ee53f769379.patch";
-        sha256 = "19rdqydwjmqg25ibmsbx7lggrr9fsyjn283zgvz1wj4iyfjwp1za";
-      })
     ];
     postPatch = ''
       substituteInPlace 'rpy2/rinterface_lib/embedded.py' --replace '@NIX_R_LIBS_SITE@' "$R_LIBS_SITE"
@@ -98,7 +91,7 @@ buildPythonPackage rec {
     ];
 
     meta = {
-      homepage = "http://rpy.sourceforge.net/rpy2";
+      homepage = "https://rpy2.github.io/";
       description = "Python interface to R";
       license = lib.licenses.gpl2Plus;
       platforms = lib.platforms.unix;

--- a/pkgs/development/python-modules/rpy2/rpy2-3.x-r-libs-site.patch
+++ b/pkgs/development/python-modules/rpy2/rpy2-3.x-r-libs-site.patch
@@ -1,10 +1,10 @@
 diff --git a/rpy2/rinterface_lib/embedded.py b/rpy2/rinterface_lib/embedded.py
-index cc32b6d..3969ad0 100644
+index 1f4babbf..322363c5 100644
 --- a/rpy2/rinterface_lib/embedded.py
 +++ b/rpy2/rinterface_lib/embedded.py
-@@ -113,6 +113,16 @@ def _initr(interactive: bool = True,
-         if isinitialized():
-             return None
+@@ -118,6 +118,16 @@ def _initr(
+         if openrlib.R_HOME is None:
+             raise ValueError('openrlib.R_HOME cannot be None.')
          os.environ['R_HOME'] = openrlib.R_HOME
 +
 +        # path to libraries

